client/jetpack-cloud/sections/partner-portal/primary/assign-license/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslate } from 'i18n-calypso';
+import { ReactElement, useEffect } from 'react';
+import CardHeading from 'calypso/components/card-heading';
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import AssignLicenseForm from 'calypso/jetpack-cloud/sections/partner-portal/assign-license-form';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+
+export default function AssignLicense( { sites }: any ): ReactElement {
+	const translate = useTranslate();
+
+	useEffect( () => {
+		const layoutClass = 'layout__content--partner-portal-assign-license';
+		const content = document.getElementById( 'content' );
+
+		if ( content ) {
+			content.classList.add( layoutClass );
+
+			return () => content.classList.remove( layoutClass );
+		}
+	}, [] );
+
+	return (
+		<Main wideLayout className="assign-license">
+			<DocumentHead title={ translate( 'Assign your new License' ) } />
+			<SidebarNavigation />
+			<CardHeading size={ 36 }>{ translate( 'Assign your new License' ) }</CardHeading>
```

Comment:
New in `your new License` does not make sense when assigning an old license. I don't have strong opinions on whether we should keep it as-is or just drop the word in both cases for simplicity.

Reply:
> I'm in favor of just dropping the word "new" for now. Thoughts?

I'm onboard with this decision 👍